### ISSUE TITLE
Pip installation throws ModuleNotFoundError: No module named 'tesla_dashcam.__main__

### ISSUE THREAD

**Issue by iAnatoly** [ehendrix23/tesla_dashcam#31]
Pip installation throws ModuleNotFoundError: No module named 'tesla_dashcam.__main__.
I've fixed it with the least amount of effort for myself:
```python
- from tesla_dashcam.__main__ import main
+ from tesla_dashcam.tesla_dashcam import main
```
but maybe you are missing an import in your ```__init__.py```

Os: Ubuntu 19.04
Python 3.7

* edit: formatting, os/python info

**Comment by raushel** [comment]
Brought this issue up, fixed for 1.10: https://github.com/ehendrix23/tesla_dashcam/issues/23

**Comment by iAnatoly** [comment]
Awesome, thank you!
